### DISCUSSION TITLE
R 5.2.0 Sunbird Inquiry Building Block QA Regression Execution Status

### DISCUSSION THREAD

**Discussion by Nadiya496** [Sunbird-inQuiry/Community#42] [General Discussions]
Hello All,

Please find the testing status of R 5.2.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage:  In Progress (8.47%)
Planned Rgression Completion-Off Date: November 4, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10609)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1Vbw9bIMO_bGs4GawSqTkKfvuzDuo7R9wzQRWtBE359M/edit#gid=0)

**Execution Details:**
![Status](https://user-images.githubusercontent.com/53341543/198030228-f995d313-2e41-4c32-a18a-82dec3bb3fa8.PNG)

**Defect Dashboard:**
![Capture](https://user-images.githubusercontent.com/53341543/198030468-ddf45098-98bb-4a30-b16b-d48795eaee00.PNG)

cc: @pallakartheekreddy @Aparna-karri591 @anunagaraj @ShailandraOjha @chitranshu-keshav, @luckynvdu @rajeevsathish @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav

**Comment by Nadiya496** [comment]
Hello All,

Please find the testing status of R 5.2.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (28.23%)
Planned Rgression Completion-Off Date: November 4, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10609)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1Vbw9bIMO_bGs4GawSqTkKfvuzDuo7R9wzQRWtBE359M/edit#gid=0)

Execution Details:
![Status](https://user-images.githubusercontent.com/53341543/198308321-3438946c-7bbe-4044-b2f7-d86c4489fa9a.PNG)

**Defect Dashboard:**
![Capture](https://user-images.githubusercontent.com/53341543/198309081-29e14627-d45b-47fd-a6a9-9218ac439371.PNG)

cc: @pallakartheekreddy @Aparna-karri591 @anunagaraj @ShailandraOjha @chitranshu-keshav, @luckynvdu @rajeevsathish @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav

**Comment by Nadiya496** [comment]
Hello All,

Please find the testing status of R 5.2.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (48.39%)
Planned Rgression Completion-Off Date: November 4, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10609)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1Vbw9bIMO_bGs4GawSqTkKfvuzDuo7R9wzQRWtBE359M/edit#gid=0)

Execution Details:
![Capture](https://user-images.githubusercontent.com/53341543/198599491-1fd3d4c6-c7f8-40f3-8ba3-5dfb9368a640.PNG)

Defect Dashboard:
![Capture](https://user-images.githubusercontent.com/53341543/198600414-a0cdc4a4-f1bf-4969-a092-e83bdafcb6a2.PNG)

cc: @pallakartheekreddy @Aparna-karri591 @anunagaraj @ShailandraOjha @chitranshu-keshav, @luckynvdu @rajeevsathish @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @PoojaJManjooran

**Comment by Nadiya496** [comment]
Hello All,

Please find the testing status of R 5.2.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (70.16%)
Planned Rgression Completion-Off Date: November 4, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10609)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1Vbw9bIMO_bGs4GawSqTkKfvuzDuo7R9wzQRWtBE359M/edit#gid=0)

Execution Details:
![image](https://user-images.githubusercontent.com/53341543/199018491-651db3b0-5088-468d-b8fc-9b482e7c2e19.png)

Defect Dashboard:
![Capture](https://user-images.githubusercontent.com/53341543/199018705-d63db5ef-552f-4e7b-bbe9-3c252df8893a.PNG)

cc: @pallakartheekreddy @Aparna-karri591 @anunagaraj @ShailandraOjha @chitranshu-keshav, @luckynvdu @rajeevsathish @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @PoojaJManjooran

**Comment by Nadiya496** [comment]
Hello All,

Please find the testing status of R 5.2.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (91.94%)
Planned Rgression Completion-Off Date: November 4, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10609)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1Vbw9bIMO_bGs4GawSqTkKfvuzDuo7R9wzQRWtBE359M/edit#gid=0)


Execution Details:
![Capture](https://user-images.githubusercontent.com/53341543/199498751-dfbf8ca4-abcb-47b5-94a6-05c6599fde89.PNG)

Defect Dashboard:
![Capture](https://user-images.githubusercontent.com/53341543/199498944-bf2d4aec-1860-4e27-9fd3-a56df0eab69b.PNG)
![Capture](https://user-images.githubusercontent.com/53341543/199499208-018cc23e-0e74-4fa2-a2ef-9714a0f27a9b.PNG)

cc: @pallakartheekreddy @Aparna-karri591 @anunagaraj @ShailandraOjha @chitranshu-keshav, @luckynvdu @rajeevsathish @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @PoojaJManjooran

**Comment by Aparna-karri591** [comment]
Hello All,

Please find the testing status of R 5.2.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (99.40%)
Planned Rgression Completion-Off Date: November 4, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10609)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1Vbw9bIMO_bGs4GawSqTkKfvuzDuo7R9wzQRWtBE359M/edit#gid=0)

Execution Details:

![image](https://user-images.githubusercontent.com/106593060/199723002-c29d144a-1f1d-4601-9ce7-7ee37237f174.png)

Defect Dashboard:
![image](https://user-images.githubusercontent.com/106593060/199723361-99c9818b-cc80-4826-a558-63177f4af052.png)

![image](https://user-images.githubusercontent.com/106593060/199724391-bccf4575-7d38-4418-bb75-24ba76fbecb1.png)



cc: @pallakartheekreddy @Nadiya496  @anunagaraj @ShailandraOjha @chitranshu-keshav, @luckynvdu @rajeevsathish @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @PoojaJManjooran
